dockstore-webservice/pom.xml

```diff
@@ -1330,7 +1331,7 @@
                     <!-- DOCKSTORE-2428 - demo how to add new workflow language -->
                     <!--  <inputSpec>https://raw.githubusercontent.com/dockstore/tool-registry-service-schemas/2.0.0-beta.5/src/main/resources/swagger/ga4gh-tool-discovery.yaml</inputSpec> -->
                     <!--  Change "serviceType.setVersion("2.0.1");" in ServiceInfoApiServiceImpl.java also -->
-                    <inputSpec>https://raw.githubusercontent.com/dockstore/tool-registry-service-schemas/2.0.5_GALAXY/openapi/ga4gh-tool-discovery.yaml</inputSpec>
+                    <inputSpec>https://raw.githubusercontent.com/dockstore/tool-registry-service-schemas/GALAXY2.0.1/openapi/ga4gh-tool-discovery.yaml</inputSpec>
```

Comment:
This should be a tag, I think (it may be actually, I think there are a tag and a branch with the same name).

Reply:
Wonder if we need to renumber these because 2.0.5 "seems" later than the official 2.0.1 TRS that we're still working on. On the other hand, `GALAXY2.0.1` doesn't seem like semantic versioning at all. 
Maybe you could abuse the prelease fields https://semver.org/#spec-item-11 to make it `2.0.1-(something)`

Reply:
Yes there is both a branch named GALAXY2.0.1 and a tag named GALAXY2.0.1, but I can't seem to get the GALAXY2.0.1 tag to display, only the branch. I should tag my PR commit for Snakemake and reference that from the webservice pom. How about '2.0.1-smk-galaxy'? Should there be an alpha or beta in the tag?

Reply:
Yeah, I could work with that.  Curious about others
Do document our reasoning somewhere where we can find it next time

Reply:
Created tag 2.0.1-beta.1-smk-galaxy on branch GALAXY2.0.1